app/assets/javascripts/quby/answers/validation.js

```diff
@@ -189,6 +208,48 @@
     return !failed;
   };
 
+  function parsePartialDate(inputs) {
+    var value = function(placeholder) {
+      var val = inputs.filter("[placeholder=" + placeholder + "]").first().val();
+      if(val === undefined || val == "")
```

Comment:
Ik dacht dat hij onterecht door de validatie zou komen. Aangezien een ongeldige waarde NaN werd en dan verderop in de default veranderd (dus de leeg test zegt 'niet leeg' en de waarde ophaal zegt 'NaN' en de default-code zegt 'maak ik er 2000 van'. Maar er is een test voor, dus blijkbaar niet.